spec/providers/coreIntegration/udpsocket.integration.src.js

```diff
@@ -27,14 +27,13 @@ module.exports = function(provider, setup) {
       return did.bind(null, markTask(name));
     };
 
-    var LOCALHOST = '127.0.0.1';
+    var LOCALHOST_V4 = '127.0.0.1';
     var checkSocketInfo = function(socketToCheck, port) {
       var getInfoTask = markTask('getInfo');
       socketToCheck.getInfo(function(state) {
-        expect(state).toEqual(jasmine.objectContaining({
-          'localAddress': LOCALHOST,
-          'localPort': port
-        }));
+        // On Chrome, this is "127.0.0.1".  On Firefox it's "localhost".
```

Comment:
It's definitely possible, but I think it's not a very small change, because the entire spy API appears to be incompatible.  Do you want to have that be part of this pull request?